src/TransitionConfigs/CardStyleInterpolators.tsx

```diff
@@ -81,6 +81,8 @@ export function forModalPresentationIOS({
   layouts: { screen },
 }: CardInterpolationProps): CardInterpolatedStyle {
   const topOffset = 10;
+
+  // TODO: update this for safe-area-context
   const statusBarHeight = getStatusBarHeight(screen.width > screen.height);
```

Comment:
this seems a bit tricky because the inset data is available and updated via context. what do you think @satya164?

Reply:
@brentvatne can we pass this data in `layouts.statusBar`

Reply:
maybe we want to call it safeAreas actually

Reply:
sounds good